cms-publish-rest/src/main/java/se/simonsoft/cms/publish/rest/PublishResource.java

```diff
@@ -223,7 +231,39 @@ public Response getReleaseJson(@QueryParam("item") CmsItemIdArg itemId,
 		return response;
 	}
 	
-	
+	/**
+	 * @param itemId
+	 * @param 
+	 * @return JSON containing started execution ID and presigned urls (same as webhook).
+	 * @throws Exception
+	 */
+	@POST
+	@Path("api/start")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response doStartApi(@QueryParam("item") CmsItemIdArg itemId, String body) throws Exception {
+		
+		logger.debug("Start publication requested with item: {} and options: '{}'", itemId, body);
+		
+		// Deserialize the body.
+		ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Never instantiate ObjectMapper, kills performance. 

We always inject ObjectReader / ObjectWriter.

Reply:
Oh, didn't know. Done.